cfme/tests/containers/test_ad_hoc_matrices.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from utils import testgen
+from utils.log import logger
+from cfme.containers.provider import ContainersProvider
+from utils.version import current_version
+from cfme.web_ui import toolbar
+from utils.appliance.implementations.ui import navigate_to
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.8"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
+
+
+@pytest.fixture
+def matrics_up_and_running(provider):
+    import requests
+
+    router = [router for router in provider.mgmt.o_api.get('route')[1]['items'] if
+              router["metadata"]["name"] == "hawkular-metrics"].pop()
+    hawkular_url = router["status"]["ingress"][0]["host"]
+    response = requests.get("https://{url}:443".format(url=hawkular_url), verify=False)
+
+    assert response.ok, "hawkular failed to start!"
+    logger.info("hawkular started successfully")
+
+
+def is_ad_hoc_greyed(provider_object):
+    navigate_to(provider_object, 'Details')
+    return toolbar.is_greyed('Monitoring', 'Ad hoc Metrics')
+
+
+@pytest.mark.polarion('CMP-10643')
+def test_ad_hoc_metrics_overview(provider,
+                                 matrics_up_and_running):
```

Comment:
on previous version i had other parameters and the line was too long.
any way i fixed this